### PR TITLE
feature: Terms and Privacy modals

### PR DESCRIPTION
**Pull request by gbarkhatov**
- changes terms
  - terms can open up privacy
- adds privacy policy

@vitsalis @jeremy-babylonlabs @jrwbabylonlab please carefully check the text content

@jeremy-babylonlabs please additionally check out any stylings inconsistencies

### CODE REVIEW

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 1]
```diff
@@ -1,798 +1,1002 @@
...
+        not receive such fees and has no ability to reverse or refund any
+        amounts paid in error.{" "}
+        <strong>
+          If you lock your bitcoin in a staking transaction without the
+          necessary amount of bitcoin to pay for the unbonding and withdrawal
+          transactions, your bitcoin will be forever locked
+        </strong>
+        . Provider will have no liability for any difference between the amount
+        of the estimate and the actual network fee, or for you locking your
+        bitcoin with insufficient bitcoin to pay for the unbonding and
```
[the remark this answers is not in the document]
Comment (vitsalis) [reply src/app/components/Modals/Terms/data/terms.tsx:160]: We use the typical (for us) convention of referring to `bitcoin` as the asset non-capitalized, while `Bitcoin` the network capitalized.

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 2]
```diff
@@ -1,798 +1,1012 @@
...
+          agency or financial regulatory authority or organization. Neither
+          Provider nor any individual Interface contributor is registered as a
+          broker, dealer, advisor, transfer agent or other intermediary.
         </li>
       </ol>
       <br />
-      <h4 className="font-bold text-primary">11. Certain risks</h4>
+      <h3>5. NO WARRANTY, NO REPRESENTATIONS</h3>
+      <br />
+      <p>
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:475]: This entire paragraph should be bold as in the doc

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 3]
```diff
@@ -1,798 +1,1012 @@
...
-        There is no business plan or revenue model for the Interface. The
-        Interface maintainers do not have revenues or a viable long-term
-        business plan, and may become unable or unwilling to fund the
-        operational costs of the Interface on a long-term basis or to fund the
-        upgrade costs required to keep the Interface up to date with current and
-        upcoming technologies.
+        Please refer to our{" "}
+        <span
+          className="text-primary underline cursor-pointer"
+          onClick={openPrivacy}
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:518]: This doesn't work.

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 4]
```diff
@@ -1,798 +1,1012 @@
...
+        >
+          [Privacy Policy]
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:520]:
```suggestion
          Privacy Policy
```

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 5]
```diff
@@ -1,798 +1,1012 @@
...
+      <h3>10. LIMITATION OF LIABILITY</h3>
       <br />
       <p>
-        Blockchain technologies and digital assets are subject to many legal and
-        regulatory uncertainties, and the Protocol or any tokens or blockchains
-        could be adversely impacted by one or more regulatory or legal
-        inquiries, actions, suits, investigations, claims, fines, or judgments,
-        which could impede or limit the ability of User to continue the use and
-        enjoyment of such assets and technologies.
+        Except to the extent otherwise required by applicable law, under no
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:606]: This paragraph should be bold as in the document

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 6]
```diff
@@ -1,798 +1,1012 @@
...
+        limitations apply even if any of the events or circumstances giving rise
+        to such damages were foreseeable and even if the Provider Parties were
+        advised of or should have known of the possibility of such losses or
+        damages and notwithstanding any failure of essential purpose of any
+        limited remedy.
       </p>
       <br />
-      <h5 className="text-primary">11.4 No Warranty</h5>
+      <p>
+        Except to the extent otherwise required by applicable law, if,
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:622]: Same here

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 7]
```diff
@@ -1,798 +1,1012 @@
...
-        your own risk. There is no representation or warranty that access to the
-        Interface will be continuous, uninterrupted, timely, or secure; that the
-        information contained in the Interface will be accurate, reliable,
-        complete, or current, or that the Interface will be free from errors,
-        defects, viruses, or other harmful elements. No advice, information, or
-        statement made in connection with the Interface should be treated as
-        creating any warranty concerning the Interface. There is no endorsement,
-        guarantee, or assumption of responsibility for any advertisements,
-        offers, or statements made by third parties concerning the Interface.
+        THE FOREGOING LIMITATIONS WILL APPLY EVEN IF THE ABOVE STATED REMEDY
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:632]: Same here

File: src/app/components/Modals/Terms/data/terms.tsx [hunk 8]
```diff
@@ -1,798 +1,1012 @@
...
-        sophisticated in using and evaluating blockchain and related
-        technologies and assets, including blockchains, tokens, and proof of
-        stake smart contract systems. The Users declare that they have conducted
-        their own thorough independent investigation and analysis of the
-        Protocol and the other matters contemplated by these Terms, and have not
-        relied upon any information, statement, omission, representation, or
-        warranty, express or implied, written or oral, made by or on behalf of
-        Interface maintainers in connection therewith, except as expressly set
-        forth in these Terms.
+        Please read the arbitration provisions in this Section 16 of the Terms
```
Comment (vitsalis) [inline src/app/components/Modals/Terms/data/terms.tsx:761]: This paragraph should be bold

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx:162]: What's the decision there?

File: src/utils/delegations/.signWithdrawalTx.ts.swp
Comment (gbarkhatov) [inline src/utils/delegations/.signWithdrawalTx.ts.swp:1]: @vitsalis .swp files are added

**Review by vitsalis — COMMENTED** [review]
terms review

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 9]
```diff
@@ -0,0 +1,693 @@
...
+    <div className="rounded-2xl border border-neutral-content p-6 dark:border-neutral-content/20 overflow-y-scroll no-scrollbar max-h-[100vh] terms-privacy-container">
+      <p className="italic">Effective as of 20 August 2024</p>
+      <br />
+      <p>
+        Babylon Labs Ltd (<strong>“Labs”</strong>, <strong>“we”</strong>,{" "}
+        <strong>“us”</strong> or <strong>“our”</strong>) is committed to
+        protecting the privacy of our users, stakeholders, and business
+        partners. This Privacy Policy describes how we use, disclose, and
+        safeguard any personal information that we collect through our digital
+        or online properties or services (including as applicable, our website,
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx]: Resolved

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 4 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx]: Done

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 3 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx]: Removed

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 5 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx]: Done

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 7 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx]: Done

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 6 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx]: Done

File: src/app/components/Modals/Terms/data/terms.tsx (hunk 8 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Terms/data/terms.tsx]: Done

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 10]
```diff
@@ -0,0 +1,696 @@
...
+        or downloading any of the Service, you agree and consent to the
+        collection, use, protection, and disclosure of your information as
+        described in this Policy. If you do not agree to this Policy, please do
+        not use, access, connect to, or download any of the Service.
+      </p>
+      <br />
+      <h3>Personal Information We Collect</h3>
+      <br />
+      <p>We may collect the following categories of personal information:</p>
+      <ol>
```
Comment (vitsalis) [inline src/app/components/Modals/Privacy/data/privacy.tsx:36]: doc uses uppercase letters for the `<ol>`

**Review by vitsalis — COMMENTED** [review]
privacy notice review

File: out.json
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply out.json:1]: It comes from @vitsalis commit, I guess some sort of setup env helper

File: src/app/components/Modals/Privacy/data/privacy.tsx (hunk 10 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx:36]: Yep, but in Terms doc also uses the mix of capitalize and uppercase headers. I chose the formatting we have in Terms.

File: out.json
[the remark this answers is not in the document]
Comment (vitsalis) [reply out.json:1]: pre-commit hook adds everything in my local -_-

[the remark this answers is not in the document]
Comment (vitsalis) [reply out.json:1]: You can safely remove

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 11]
```diff
@@ -0,0 +1,696 @@
...
+        </li>
+      </ul>
+      <br />
+      <h3>How We Use Your Personal Information</h3>
+      <br />
+      <p>
+        We may use your personal information for the following purposes or as
+        otherwise described at the time of collection:
+      </p>
+      <ul className="list-disc list-inside">
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx:216]: Done

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 12]
```diff
@@ -0,0 +1,696 @@
...
+          personal information to:
+          <ul className="list-disc list-inside">
+            <li>administer promotions and contests;</li>
+            <li>
+              communicate with you about promotions or contests in which you
+              participate;
+            </li>
+            <li>
+              contact or market to you after collecting your personal
+              information at an event;
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx]: But it should end with that stuff? ";"

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 13]
```diff
@@ -0,0 +1,696 @@
...
+        to your identity, or in order to locate the information sought, and we
+        are not obliged to provide you with personal data or otherwise address
+        your request unless you satisfy our requirements. We will endeavour to
+        respond to a written request within the timelines set out in the British
+        Virgin Islands DPA.
+      </p>
+      <br />
+      <h3>Your Choices</h3>
+      <br />
+      <ul className="list-disc list-inside">
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx:534]: Done

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 14]
```diff
@@ -0,0 +1,696 @@
...
+          prevent web beacons from functioning.
+        </li>
+        <li>
+          <strong>Do Not Track</strong>. Some Internet browsers may be
+          configured to send &quot;Do Not Track&quot; signals to the online
+          services that you visit. We currently do not respond to &quot;Do Not
+          Track&quot; signals. To find out more about &quot;Do Not Track&quot;,
+          please visit{" "}
+          <a
+            href="https://http://www.allaboutdnt.com"
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx]: Done

File: src/app/components/Modals/Privacy/data/privacy.tsx [hunk 15]
```diff
@@ -0,0 +1,696 @@
...
+          <strong>Declining to provide information</strong>. We need to collect
+          personal information to provide certain services. If you do not
+          provide the information we identify as required or mandatory, we may
+          not be able to provide those services.
+        </li>
+        <li>
+          <strong>Remove or anonymize your content</strong>. You may request
+          that the content that you posted to the Service be removed or
+          anonymized such that your personal information will not be publicly
+          identifiable on the Service by contacting us. We may decline to
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx]: Done

File: src/app/components/Modals/Privacy/data/privacy.tsx (hunk 12 above)
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx]: Done

File: out.json
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply out.json:1]: Removed

File: src/app/components/Modals/Privacy/data/privacy.tsx (hunk 10 above)
Comment (vitsalis) [reply src/app/components/Modals/Privacy/data/privacy.tsx:36]: ok

**Review by vitsalis — APPROVED** [review]
content lgtm

File: src/app/components/Modals/Privacy/data/privacy.tsx (hunk 10 above)
Comment (gbarkhatov) [reply src/app/components/Modals/Privacy/data/privacy.tsx]: Done

**Pull request merged**